main/postgres/initdb.d/README.markdown

```diff
@@ -0,0 +1,14 @@
+This directory helps you prepare your Postgres instance for MPortal to run.
+
+Context: when MPortal runs for the first time, it will create all the Postgres tables it needs;
+but to do that, it assumes that certain roles/"schemas" are already set up in Postgres.
+
+Here are instructions for how to take a fresh Postgres instance and make it ready for MPortal to run for the first time:
+
+- First, create two users (probably by logging in as the root user via PGAdmin), named `metrics_dba` and `akka_dba`.
+- `export POSTGRES_USER=postgres` (or whatever your root user's name is)
+- `export POSTGRES_DB=????` (TODO(spencerpearson): what is this? Our production Postgres seems to have databases named `postgres` and `rdsadmin`, plus `metrics` which these scripts seem to create.)
```

Comment:
* `rdsadmin` is created by AWS on your RDS instance
* `postgres` is the default Postgresql database

We could have easily used the default database, but elected not to, and instead create a new one called `metrics` and within it two schemas: `akka` and `portal`. The former being used by Metrics Portal's distributed actor system (when such features are enabled) and the latter as SQL data source for features such as host registry, reports, alerts (soon).

This is not that different from what happens locally (intentionally). The Docker PG instance starts-up and creates an empty `postgres` database as defined by the environment variable set on the docker image in the `pom.xml` file. Then the init script is run which creates the primary root account, and then additional users and roles and the metrics database.